app/controllers/training/pages_controller.rb

```diff
@@ -19,7 +19,7 @@ def index
     end
 
     def show
-      if content.is_question?
+      if content.is_question? || content.feedback_question?
```

Comment:
We are trying to evolve the CMS content away from "questionnaires" but what that meant is we could match on part of that word.

We now want to update the following rather than add on an extra call to content.

```ruby
  def is_question?
    page_type.match?(/formative|summative|confidence|feedback/)
  end
```